scripts/winresources/event_messages.mc

```diff
@@ -0,0 +1,39 @@
+MessageId=1
```

Comment:
Actually, I'm not sure these are needed in the CLI. They are for event logs for when the daemon is running as a service.